.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
what is this file? Let's remove this and also .yarn/releases/yarn-1.19.0.cjs and .yarn/releases/yarn-1.22.19.cjs
remember review your own PRs